### PR TITLE
restaurant independent challenge

### PR DESCRIPTION
**Pull request by jordanfbeck0528**
Yikes. No idea how to  how to pass `test_it_has_closing_time` for iteration 2.

### CODE REVIEW

File: lib/restaurant.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  def initialize(opening_time, name)
+    @opening_time = opening_time
+    @name = name
+    @dishes = []
+
+  end
+
+  def dishes
+    @dishes
+  end
```
Comment (KathleenScriver) [inline lib/restaurant.rb:15]: this does work to read back the dishes, but how can you write this more Ruby-like? Hint, line 5.

File: lib/restaurant.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  # def closing_time(time)
+  #   @opening_time += time.to_s
+  # end
+
+  def add_dish(dish)
+    @dishes << dish
```
Comment (KathleenScriver) [inline lib/restaurant.rb:22]: Great job.

**Review by KathleenScriver — COMMENTED** [review]
Great start here. I encourage you to set up sometime within the next day or two to meet with Krista and your mentor to walk through this challenge.

File: lib/restaurant.rb (hunk 1 above)
Comment (jordanfbeck0528) [reply lib/restaurant.rb:15]: Fixed. Added `:dishes` to attr_reader in line 5

File: lib/restaurant.rb (hunk 2 above)
Comment (jordanfbeck0528) [reply lib/restaurant.rb:22]: Thanks!

**Pull request closed without merging**